contracts/simple_airdrop/src/contract.rs

```diff
@@ -131,7 +131,6 @@ pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
     match msg {
         QueryMsg::Config {} => to_binary(&query_config(deps)?),
         QueryMsg::State {} => to_binary(&query_state(deps)?),
-        QueryMsg::HasUserClaimed { address } => to_binary(&query_user_claimed(deps, address)?),
```

Comment:
Why?

Reply:
i updated the logic a bit so we can update the merkle roots if need be so a user can claim airdrop multiple times, in which case this query doesn't make sense

Reply:
gotcha